svm/src/transaction_processor.rs

```diff
@@ -386,82 +385,77 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
         pubkey: &Pubkey,
```

Comment:
The rustdocs for this method are 100% off? Could you update to say something like

```
Loads the program with the given pubkey.

If the account doesn't exist it returns `None`. If the account does exist, it must be a program
account (belong to one of the program loaders). Returns Some(InvalidAccountData) if the program
account is Closed, contains invalid data or any of the ancillary program accounts contain invalid
data.
```